src/patternfly/components/Dropdown/styles.scss

```diff
@@ -3,7 +3,7 @@
 .pf-c-dropdown {
   // Toggle
   --pf-c-dropdown__toggle--PaddingTop: var(--pf-global--spacer--xs);
-  --pf-c-dropdown__toggle--PaddingRight: var(--pf-global--spacer--md);
+  --pf-c-dropdown__toggle--PaddingRight: var(--pf-global--spacer--sm);
```

Comment:
Is this change intentional? Did the visual design for dropdowns change?

This is an example of the Dropdown before this change:
![image](https://user-images.githubusercontent.com/21063328/42397068-6d8c7458-8132-11e8-959d-1f24f782b3a3.png)
And after:
![image](https://user-images.githubusercontent.com/21063328/42397079-79b3d320-8132-11e8-87cb-4607c045e2f3.png)

Reply:
yes, it changed when @mceledonia updated the forms.